### ISSUE TITLE
Clarifying GLUE Benchmark Accuracy: Validation or Test Set?

### ISSUE THREAD

**Issue by monk1337** [jiaweizzhao/GaLore#13]
Hello, I enjoyed reading your excellent paper. I have one question: is the accuracy mentioned for the GLUE benchmarks based on the validation set or the test set? The paper does not specify this detail. Thank you for your clarification.

**Comment by Kyriection** [comment]
Hi, Thanks for your interest in our work. The results of GLUE benchmarks are reported on the validation set, following the implementation from https://github.com/huggingface/transformers/blob/main/examples/pytorch/text-classification/run_glue.py.
